tests/conftest.py

```diff
@@ -334,6 +334,20 @@ def client(testconfig, iaas, imageurl, request) -> Iterator[RemoteClient]:
         raise ValueError(f"invalid {iaas=}")
 
 
+@pytest.fixture(autouse=True)
+def test_features(client, request):
+    (exit_code, output, error) = client.execute_command("cat /etc/os-release", quiet=True)
```

Comment:
We should add a documentation text to this fixture explaining what it does and why it is needed.